src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -1,28 +1,74 @@
 /* eslint-disable no-underscore-dangle */
-import * as React from 'react';
+import {useNavigation} from '@react-navigation/native';
 import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and a duplicate import removed. This improves the readability of the code.
```diff
- import * as React from 'react';
- import PropTypes from 'prop-types';
+ /* eslint-disable no-underscore-dangle */
+ import {useNavigation} from '@react-navigation/native';
+ import PropTypes from 'prop-types';
+ import * as React from 'react';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->